### ISSUE TITLE
Delete Container: Labels appearing text are not defined in <label> and are also not associated with the edit fields.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1039]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1241119

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section and select any of the item then some new menu item appears in the top like New SQL Query, New Stored Procedure..
Navigate to "Delete Collection" menu item from the more options(...) and select it.
Navigate through all the controls under "Delete  Container" blade.
Navigate to 'Confirm by typing the container ID' required edit field.
Inspect the Edit text box through developer tool F12.
Verify the issue.

Actual Result:
"Confirm by typing the container ID" present under 'Delete container' blade are not define within label tag and is also not associated with the edit field present below it due to which screen reader does not announce its information. 

Expected Result:
"Confirm by typing the container ID" present under 'Delete container' blade should be defined within label tag and should also be associated with the the edit field present below it so that  screen reader announce its information. 

Note:
Similar issue exists for all edit fields present under 'Delete Container' blade.
This issue also exists for edit fields of "Delete Database" dialog. Refer to the Attachment: [MAS3.3.2-Label appearing text is not associated.]
This issue exists throughout the application for similar controls.

![image](https://user-images.githubusercontent.com/8519407/132718619-eb67ad8a-2d8e-43bd-a584-44816f6f6854.png)

![image](https://user-images.githubusercontent.com/8519407/132718638-0f2b9b33-ffa3-49f2-988f-6561a3ea5dc8.png)
